api/rpc/stat/stat.go

```diff
@@ -29,9 +29,16 @@ func New(cfg Config) (*Stat, error) {
 	return &Stat{conn: c}, err
 }
 
-// ExecuteQuery implements business logic for StateServer interface
+// ExecuteQuery implements business logic for StatServer interface
```

Comment:
This is the interface generated from the "make proto" step, so it exists with the stat.pb.go module